packages/core/src/command/consolidated/search/fhir-resource/search-consolidated.ts

```diff
@@ -320,12 +317,16 @@ export async function hydrateMissingReferences({
   if (missingRefIds.length < 1) return resources;
 
   const uniqueIds = uniq(missingRefIds);
+  const filteredUniqueIds =
+    idsToExclude.length > 0
+      ? uniqueIds.filter(uniqueId => !idsToExclude.some(idToExclude => idToExclude === uniqueId))
```

Comment:
nit, use includes over some for trivial cases.

(uniqueId) => !idsToExclude.includes(uniqueId)

Reply:
good call, done